app/components-react/sidebar/NavTools.tsx

```diff
@@ -91,64 +94,159 @@ export default function SideNav() {
+                }
+                onTitleClick={() => {
+                  !isOpen && throttledOpenDashboard();
+                  expandMenuItem(ENavName.BottomNav, menuItem.title as EMenuItem);
+                }}
+                className={styles.badgeScale}
+              >
+                {menuItem?.subMenuItems.map((subMenuItem: IMenuItem) => (
+                  <Menu.Item
+                    key={subMenuItem.key}
+                    title={$t(subMenuItem.title)}
+                    className={cx(!isOpen && menuItem.isExpanded && styles.hideSubMenu)}
+                    onClick={() => throttledOpenDashboard(subMenuItem?.type)}
+                  >
+                    {$t(subMenuItem.title)}
+                  </Menu.Item>
+                ))}
+              </Menu.SubMenu>
+            );
+          } else if (menuItem.title === EMenuItem.GetHelp) {
+            return (
+              <Menu.Item
+                key={menuItem.key}
+                title={$t(menuItem.title)}
+                icon={
+                  <div>
+                    <Badge count={<i className={cx('icon-pop-out-3', styles.linkBadge)} />}>
+                      <i className={menuItem?.icon} />
+                    </Badge>
+                  </div>
+                }
+                onClick={() => openHelp()}
+              >
+                {$t(menuItem.title)}
+              </Menu.Item>
+            );
+          } else if (menuItem.title === EMenuItem.Settings) {
+            return (
+              <Menu.Item
+                key={menuItem.key}
+                title={$t(menuItem.title)}
+                icon={<i className={menuItem?.icon} />}
+                onClick={openSettingsWindow}
+              >
+                {$t(EMenuItem.Settings)}
+              </Menu.Item>
+            );
+          } else if (menuItem.title === EMenuItem.Login) {
+            return (
+              <Menu.Item
+                key={menuItem.key}
+                title={!isLoggedIn ? $t(EMenuItem.Login) : $t('Log Out')}
+                className={cx(styles.login, !isOpen && styles.loginClosed)}
+                icon={!isOpen && <i className="icon-user" />}
+                onClick={() => handleAuth()}
+              >
+                {!isLoggedIn ? (
+                  <Typography.Text underline style={{ marginBottom: '0px', flexGrow: 1 }}>
+                    {$t(EMenuItem.Login)}
+                  </Typography.Text>
+                ) : (
+                  isOpen && (
+                    <>
+                      {platform && (
+                        <PlatformLogo
+                          platform={platform?.type!}
+                          className={cx(
+                            styles.platformLogo,
+                            styles[`platform-logo-${platform?.type ?? 'default'}`],
+                          )}
+                        />
+                      )}
+                      <Typography.Text className={styles.username}>
```

Comment:
what's the purpose of this element?

Reply:
This is so I can target the text to `flex-grow` it so that the login arrow shows at the right side of the container. I opted to use an antd element but could use just a div.

Reply:
gotcha-- yea i think if the purpose is just to hold a css class it's probably better to keep things simple and just use a `span` or `div`, since we don't necessarily know what antd components are doing under the hood and could lead to unexpected or unperformant behavior

i think a good rule of thumb is if you're using an antd component you should have a present and clear reason to do so, like the `Menu` component for example

Reply:
Thanks! Will refactor